voice/transfer-call-inline-ncco.py

```diff
@@ -8,30 +8,21 @@
 envpath = join(dirname(__file__), './.env')
 load_dotenv(envpath)
 
-# Init the client
-print(os.getenv("NEXMO_PRIVATE_KEY"))
+# Init the nexmo client
 client = nexmo.Client(
     application_id=os.getenv('NEXMO_APPLICATION_ID'),
     private_key=os.getenv("NEXMO_PRIVATE_KEY")
 )
 
-response = client.create_call({
-    "to": [{"type": "phone", "number": os.getenv('TO_NUMBER')}],
-    "from": {"type": "phone", "number": os.getenv('FROM_NUMBER')},
-    "ncco": [
-        {
-            "action": "talk",
-            "text": "This is just a text whilst you tranfer to another NCCO"
-        }
-    ]
-})
+UUID = os.getenv('CALL_UUID')
```

Comment:
env var should match the var name be called `UUID`

Reply:
This is a standard for 5 code snippets I'm just following after what is already done.